includes/Util.php

```diff
@@ -26,15 +26,18 @@ function generateDomainId() {
 	return substr( $domain_id_full, 0, 10 );
 }
 
-function getDomainId(): string {
+function getDomainID(): string {
+	return MediaWikiServices::getInstance()->getConfigFactory()->makeConfig( 'da' )
+		->get( 'DomainID' );
+}
+
+function maybeGenerateDomainId(): void {
```

Comment:
Global functions like these are basically impossible to unit test in PHP since you can't money-patch.